help/sources/tutorials/ui/create/marketing-automation/braze.md

```diff
@@ -0,0 +1,58 @@
+title: Create a Braze Source Connection in the UI
+type: Tutorial
+description: Learn how to create a Braze source connection using the Adobe Experience Platform UI and Braze Currents
+exl-id: 452b7290-b9e8-4728-8b58-0e0c76bd9449
+---
+# Create a [!DNL Braze] source connection in the UI
+
+Source connectors in Adobe Experience Platform provide the ability to ingest externally sourced data on a scheduled basis. This tutorial provides steps for creating a streaming [!DNL Braze] source connector using the [!DNL Platform] user interface.
+
+## Getting started
+
+This tutorial requires a working understanding of the following components of Adobe Experience Platform:
+
+*   [[!DNL Experience Data Model (XDM)] System](../../../../../xdm/home.md): The standardized framework by which [!DNL Experience Platform] organizes customer experience data.
+    *   [Basics of schema composition](../../../../../xdm/schema/composition.md): Learn about the basic building blocks of XDM schemas, including key principles and best practices in schema composition.
+    *   [Schema Editor tutorial](../../../../../xdm/tutorials/create-schema-ui.md): Learn how to create custom schemas using the Schema Editor UI.
+*   [[!DNL Real-Time Customer Profile]](../../../../../profile/home.md): Provides a unified, real-time consumer profile based on aggregated data from multiple sources.
+
+This tutorial also requires a working understanding of [[!Braze Currents]https://www.braze.com/docs/user_guide/data_and_analytics/braze_currents).
+
+If you already have a [!DNL Braze] connection, you may skip the remainder of this document and proceed to the tutorial on [configuring a dataflow](../../dataflow/marketing-automation.md).
+
+## Create your [!DNL Platform] Data Source
+
+Log in to [Adobe Experience Platform](https://platform.adobe.com) and then select **[!UICONTROL Sources]** from the left navigation bar to access the **[!UICONTROL Sources]** workspace. The **[!UICONTROL Catalog]** screen displays a variety of sources from which you can select.
+
+Select the Marketing Automation category from the catalog on the left-hand side of your screen. Alternatively, you can find the specific source you wish to work with using the search option.
+
+Under the **[!UICONTROL Marketing automation]** category, select **[!UICONTROL Braze]**. If this is your first time using this connector, select **[!UICONTROL Configure]**. Otherwise, select **[!UICONTROL Add data]** to create a new [!DNL Braze] connector.
+
+![catalog](../../../../images/tutorials/create/braze/catalog.png)
+
+The **[!UICONTROL Connect to Braze]** page appears. Upload the provided Braze Currents sample file. This sample file contains all possible fields that Braze might send as part of an event.  Proceed to create a new data set. Map your data set into your schema. Note that Braze timestamps are in seconds, and that you'll need to create calculated fields in milliseconds to properly refelect the timestamps in {!DNL Platform].
```

Comment:
We can add screenshots here for the steps: Select Data, Dataflow Detail, and Mapping. But as an alternative we can break this down into steps as well.

Next, you must:
- Upload the provided [!DNL Braze Currents] sample file... this file contains all possible fields that Braze might send as part of an event.(can we add more detail on this provided sample file? From who/where does a customer get the file?)
- "Provide dataflow details, including information on your dataset and the schema that you are mapping to."
- Configure mapping for your data. **Note**: Braze timestamps are measured in seconds and not milliseconds. Therefore, you must create calculated fields in milliseconds to properly reflect the timestamps in Experience Platform (please add an example of the calculated field formula as well, I think it is something like "time * 1000"

Reply:
Breaking down into steps is a great idea. Is the sample file typically hosted in AEP docs or partner (Braze)? I'm fine either way, just let me know.

And yeah screenshots are a good idea. It wasn't clear to me if it was expected that I would upload the AEP screenshots or if that was something Adobe would handle. Can you please advise?

Reply:
- Re: sample file: I think we can make the call on this one. I think it would be good to be hosted in both sides. What do you think?
- Re: screenshots: Partner generally uploads the screeenshots since I may sometimes not have access to the source being integrated. I can make updates for maintenance though, once they are live. We only only one screenshot for each of the three steps. Please use this hex code too if possible: 19E641. Thanks!

Reply:
Where/how would you like me to use that hex code?

Reply:
For the markups on screenshots. Ideally, we have the "actions" marked up. Similar to: https://experienceleague.adobe.com/docs/experience-platform/sources/ui-tutorials/on-demand-ingestion.html?lang=en 

I can also mark up the screenshots once I port this PR over to the internal repo. I just don't have access to an org with Braze Currents source yet and can't get the screenshots until after we release

Reply:
If it's all the same to you on the sample file, I'd like to host it on our public examples repo. That'll make it easier to maintain.

Reply:
okay that sounds good to me!